include/fiddle/mechanics/fiber_network.h

```diff
@@ -0,0 +1,32 @@
+#ifndef included_fiddle_postprocess_fiber_network_h
+#define included_fiddle_postprocess_fiber_network_h
+
+#include <deal.II/grid/tria.h>
+#include <deal.II/base/array_view.h>
+#include <deal.II/base/table.h>
+
+namespace fdl
+{
+  using namespace dealii;
+    
+    template <int dim, int spacedim = dim>
+    class FiberNetwork

```

Comment:
```suggestion
    /**
     * xx yy zz
     */
    template <int dim, int spacedim = dim>
    class FiberNetwork
```
We should write some basic documentation explaining what this class does.